pkg/config/config_template.yaml

```diff
@@ -256,4 +256,34 @@ metadata_collectors:
 #
 # URL where the ECS agent can be found. Standard cases will be autodetected.
 # ecs_agent_url: http://localhost:51678
-{{ end -}}
\ No newline at end of file
+{{ end -}}
+{{- if .ProcessAgent }}
+# Process agent specific settings
+#
+# process_config:
+#   enabled: true
```

Comment:
On our side this is a string which is dumb but allows is to differentiate between the default value (false) and no value with unmarshal. Is there a better approach here that I'm missing?

Reply:
We delegate this logic to Viper where we can bind default values explicitly but strings work for us, will add a comment